examples/word_of_day.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./word_of_day_template');
```

Comment:
Generally, you want the template body to be the requirement, not the code to load it.

i.e. You will probably want to put only the template into a separate file and load it inline in this one.